### PR TITLE
Remove Symfony 6 hacks

### PR DESCRIPTION
**Pull request by sidz**
Remove Symfony 6 hacks as it has been released

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -53,10 +53,8 @@ jobs:
             composer-${{ runner.os }}-
             composer-
 
-      - name: Configure Symfony v6@dev hacks
-        if: matrix.symfony-require == '^6'
-        run: |
-          composer config minimum-stability beta
+      - name: Globally install symfony/flex
```
Comment (sidz) [inline .github/workflows/ci.yaml:56]: my bad, forgot to add this section in previous PR.

**Review by maks-rafalko — APPROVED** [review]
:ok_hand: as long as the build is fixed

**Comment by maks-rafalko** [comment]
is it still relevant? :)

**Comment by sidz** [comment]
yup @maks-rafalko. I'll rebase this branch once https://github.com/infection/codeception-adapter/pull/21 merged

**Comment by sidz** [comment]
ready for review @maks-rafalko

File: .github/workflows/e2e-tests.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# yamllint disable rule:line-length
+# yamllint disable rule:braces
+
+name: E2E Tests
```
Comment (sidz) [inline .github/workflows/e2e-tests.yaml:4]: Codeception 4 is not compatible with Symfony 6. That's why I moved E2E tests to separate jobs.

As the next step I think we could drop for codeception < 5 support as well as php 7.4 (8.0 probably either)

File: .github/workflows/e2e-tests.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php-version: [ '7.4', '8.0', '8.1' ]
+        coverage-driver: [ pcov ]
+
+    name: CI with PHP ${{ matrix.php-version }}, using ${{ matrix.coverage-driver }}, with Symfony ${{ matrix.symfony-require }}
```
Comment (maks-rafalko) [inline .github/workflows/e2e-tests.yaml:22]: it's missing `symfony-require` input I guess: 

image

Comment (sidz) [reply .github/workflows/e2e-tests.yaml]: good catch. No longer needed

**Review by maks-rafalko — APPROVED** [review]

**Comment by sidz** [comment]
thanks @maks-rafalko

**Pull request merged**